### PR TITLE
its done

### PR DESCRIPTION
**Pull request by Jaikishan0511**

### CODE REVIEW

File: html/chatbox.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
+<html>
+    <head>
+            <meta name="viewport" content="width=device-width, initial-scale=1">
+        <style>
+            body{
+                width:100%;
+                height:100%;
```
Comment (Shrikant9) [inline html/chatbox.html:7]: Set the `Tab Size` in your code editor to `2` spaces instead of `4`.

File: html/Secvsdiv.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+<section>
+  <h1>WWF's Panda symbol</h1>
+  <p>The Panda has become the symbol of WWF. The well-known panda logo of WWF originated from a panda named Chi Chi that was transferred from the Beijing Zoo to the London Zoo in the same year of the establishment of WWF.</p>
+</section>
+
+<p><strong>Note:</strong> The section tag is not supported in Internet Explorer 8 and earlier versions.</p>
+</div>
+</body>
+</html>
```
Comment (Shrikant9) [inline html/Secvsdiv.html:18]: Make sure your editor adds a new line at the end of file when you save the file.

If you're using vscode, add this to the config:
`"files.insertFinalNewline": true`

File: html/display.html [hunk 3]
```diff
@@ -0,0 +1,105 @@
+<html>
+<head>
+    <style>
+#one{
+display:inline;
+background-color:red;
+
+}
+#two{
+display:block;
```
Comment (Shrikant9) [inline html/display.html:10]: Prettify all of your code. There's a utility (and usually comes as a plugin to most editors) called `Prettier`. Gets the job done. 
Do this to all of your files.

File: html/mediaquery.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+<head>
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<style>
+.text {
+
+  padding: 20px;
+  color: white;
+}
+@media only screen and (max-width:600px)/*extra small devices (phones, 600px and down)*/
```
Comment (Shrikant9) [inline html/mediaquery.html:11]: I need you to keep only 2 media queries. 
1. The default should be mobile.
2. Tablet: `@media only screen and (min-width:768px)`
2. Desktop: `@media only screen and (min-width:1200px)`

File: html/pagelayout.html [hunk 5]
```diff
@@ -1,29 +1,25 @@
...
+      position: sticky;
+      top:0px;
       background-color: black;
-     padding:10px;
+      padding:10px;
       overflow: hidden;
       
     }
-    nav a{
+    .nav a{
```
Comment (Shrikant9) [inline html/pagelayout.html:22]: Avoid using tag names here.
`.nav a` might pollute all the `a` tags in the rest of the project.
Use something like `.navlink` class and directly give it to the `a` tag instead.

File: html/pagelayout.html [hunk 6]
```diff
@@ -67,15 +69,16 @@
...
 <body>
-    <div>
-    <header>
+    <div class="header">
+        <h2>HEADER </h2>
+    </div>
             
-    <nav>
+    <div class="nav">
         <a href="">home</a>
```
Comment (Shrikant9) [inline html/pagelayout.html:77]: Instead of having an empty href, you can give `href="/"`.

**Review by Shrikant9 — CHANGES_REQUESTED** [review]
Good job at finishing the work @joker0511 
Just want some final checks to be ticked, please tend to those.

File: html/Secvsdiv.html (hunk 2 above)
Comment (Jaikishan0511) [reply html/Secvsdiv.html:18]: done

File: html/chatbox.html (hunk 1 above)
Comment (Jaikishan0511) [reply html/chatbox.html:7]: done

File: html/mediaquery.html (hunk 4 above)
Comment (Jaikishan0511) [reply html/mediaquery.html:11]: done

File: html/pagelayout.html (hunk 5 above)
Comment (Jaikishan0511) [reply html/pagelayout.html:22]: done

File: html/pagelayout.html (hunk 6 above)
Comment (Jaikishan0511) [reply html/pagelayout.html:77]: done

File: html/display.html (hunk 3 above)
Comment (Jaikishan0511) [reply html/display.html:10]: done

**Pull request merged**